src/welcome_view.html

```diff
@@ -7,10 +7,10 @@ <h1>Welcome to Ordino</h1>
         genes, cell lines, and tissue samples based on a rich set of experimental and metadata.
```

Comment:
Ordino is independent of the entity types used. In one setup it is about genes, celllines, and tissues but just because of the corresponding data provider plugin not because of Ordino itself.

Reply:
Ok, I get your point. Should the BI welcome page then located in tdp_bi_bioinfodb?

Reply:
And if I should move the page to tdp_bi_bioinfodb, what should the "default" Ordino welcome page show?